### PR TITLE
[add] unit test

### PR DESCRIPTION
**Pull request by kazukazuinaina**
Hi! @whatyouhide 

I adopt unit test in this repo.
It can syntax check about `Vim script` by `vint`.
Could you check this Pull Request?
PS: If you merge this patch, `github actions` may work.

## Reference

reviewdog : > https://github.com/reviewdog/action-vint
vint : > https://github.com/Vimjas/vint

### CODE REVIEW

**Comment by kazukazuinaina** [comment]
If you don't need this pull request, feel free to close it. 🙇

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  vint:
+    name: runner / vint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: vint
+        uses: reviewdog/action-vint@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```
Comment (whatyouhide) [inline .github/workflows/main.yml]: @kazukazuinaina where is this coming from? Should I set it up anywhere?

Comment (kazukazuinaina) [reply .github/workflows/main.yml]: @whatyouhide 
Thanks for review this Pull Request.
I hope this reference will help you.

> https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

Comment (kazukazuinaina) [reply .github/workflows/main.yml]: If we delete this config, github action may cannot running...

**Comment by kazukazuinaina** [comment]
Thank you.

**Pull request merged**